pkg/kv/db.go

```diff
@@ -679,12 +679,12 @@ func (db *DB) AddSSTable(
 ) error {
 	b := &Batch{Header: roachpb.Header{Timestamp: batchTs}}
 	b.addSSTable(begin, end, data, disallowConflicts, disallowShadowing, disallowShadowingBelow,
-		stats, ingestAsWrites, false /* writeAtBatchTS */)
+		stats, ingestAsWrites, false /* writeAtBatchTS */, hlc.Timestamp{} /* sstTimestamp */)
 	return getOneErr(db.Run(ctx, b), b)
 }
 
 // AddSSTableAtBatchTimestamp links a file into the Pebble log-structured
-// merge-tree.
+// merge-tree. All keys in the SST must have a have batchTs as their timestamp.
```

Comment:
Consider adding a note that the timestamp may get pushed by the server.

Reply:
Done.